src/toICN-after.js

```diff
@@ -1,38 +1,14 @@
 let isAutoKeyDetection = true;
 let isKeyWritten = false;
 let detectedKey = new exports.Key();
```

Comment:
```suggestion
let detectedKey;
```

右辺はもう使わなそうなので、このようにするのはどうでしょう？